core/src/main/scala/org/http4s/sbt/Http4sOrgPlugin.scala

```diff
@@ -55,7 +56,13 @@ object Http4sOrgPlugin extends AutoPlugin {
   lazy val githubActionsSettings: Seq[Setting[_]] =
     Seq(
       githubWorkflowJavaVersions := List("8", "11", "17").map(JavaSpec.temurin(_)),
+      tlCiScalafixCheck := false, // we add our own, that skips Scala 3
       githubWorkflowBuildPostamble ++= Seq(
+        WorkflowStep.Sbt(
+          List("scalafixAll --check"),
+          name = Some("Check scalafix lints"),
+          cond = Some(s"${primaryJavaCond.value} && !startsWith(matrix.scala, '3.')")
+        ),
```

Comment:
In theory, this could be annoying for any http4s project that decides to target only Scala 3. But they can also just flip `tlCiScalafixCheck := true` to get back the sbt-typelevel generated step which will run.